dev/.scalafmt.conf

```diff
@@ -22,6 +22,13 @@ align.tokens = []
 optIn = {
   configStyleArguments = false
 }
-danglingParentheses = false
-docstrings = JavaDoc
+danglingParentheses.preset = false
+docstrings.style = Asterisk
 maxColumn = 98
+runner.dialect = scala212
+fileOverride {
+  "glob:**/src/**/scala-2.13/**.scala" {
+    runner.dialect = scala213
+  }
+}
+version = 3.5.9
```

Comment:
[ERROR]
org.scalafmt.dynamic.exceptions.ScalafmtException: missing setting 'version'. To fix this problem, add the following line to .scalafmt.conf: 'version=3.2.1'.